app/controllers/books_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class BooksController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    render json: Book.all.page
```

Comment:
According to the documentation, by not adding a page, the first one is taken by default. This is a simple solution only for the first load of the index and to learn how it works, but also I can propose to receive 'page' per parameter and it will work for all cases.

Reply:
Exactly, you need to be able to handle requests to any page, not only the first one

Reply:
Ok